packages/webapp/src/utils/application.js

```diff
@@ -352,7 +352,12 @@ const application = (session, account) => {
       // BNGP-3863, change legal agreement end date to habitat enhancements end date, but leave application as legalAgreementEndDate :facepalm:
       legalAgreementEndDate: session.get(constants.redisKeys.HABITAT_ENHANCEMENTS_END_DATE_KEY),
       habitatPlanIncludedLegalAgreementYesNo: session.get(constants.redisKeys.HABITAT_PLAN_LEGAL_AGREEMENT_DOCUMENT_INCLUDED_YES_NO),
-      landowners: getLandowners(session),
+      landowners: session.get(constants.redisKeys.ANY_OTHER_LANDOWNERS_CHECKED) === 'Yes'
```

Comment:
please make this change in the getLandowners function in application.js to make the code cleaner